src/Game.cpp

```diff
@@ -146,18 +146,18 @@ void main()
 
 			size = (4 * sizeof(float)) + sizeof(double) + sizeof(bool); //calculating buffer size
```

Comment:
I thought we were moving the size calculation outside the game loop? We can put it after the network is connected.